rules.json

```
@@ -2,16 +2,167 @@
+            "• Character group limitations;", 
+            "Law Enforcement:",
+            "A. You cannot have two (2) characters that are a part of different law enforcement departments.",
+            "B. You cannot have two (2) characters in the same law enforcement department.",
+"Other Emergency Services:",
+            "A. You cannot have one character in multiple Emergency Services positions.",
+            "B. You can be in dispatch and another emergency service position.",
+"Gangs/Criminal Groups:",
+"A. You cannot have two (2) or more characters in the same gang/criminal group/organization.",
+"B. If you perm-kill your character, you cannot join that same group/gang for six (6) months on a new character.",
+"C. While in the commission of a crime, only six members of the group may be present during the crime.",
+"'Families':",
+           " a. You cannot make a whole family controlled by you, for instance, you cannot have this character be the brother, and another character that you own be another brother and/or cousin. All of your characters cannot have connections with each other, each one has a mind of its own.",
+            
+            "• If you're a part of any Emergency Services, and you end up resigning from your position, you need to wait 30 days before you can be considered for reinstatement.",
+            "• If you discover a bug, do NOT abuse/exploit it as you will be severely punished. Instead, please file a bug report on the forums so we can fix it as soon as possible.",
+            "• Do NOT have any inappropriate or NSFW profile pictures on our Discord and/or Forums.",
+"We Strictly follow the Discord TOS and Guidelines! if you are found breaking either of these, you will be BANNED without notice!",
+"TOS: https://discordapp.com/terms",
+"Guidelines: https://discordapp.com/guidelines",
+
+
+            "Policy B: Chain of Command (CoC)",
+"• Everyone will start from the bottom and may be promoted to any position if necessary.",
+"• Do NOT skip ranks.",
+"• Do NOT ask for promotions/ranks.",
+"• Directors/Administration always have the final say in any decision made.",
+
+            "Policy C: Respect Staff and Their Decisions",
+"• Staff Members always have the final say, no point in arguing a punishment. Instead, accept the consequences and if you still feel like it was treated poorly or you didn't do anything, fill out a Staff Report and appeal your punishment.",
+"• Do NOT question decisions made by the Staff Team of TXDPSRP.",
+"• If you believe there is a problem with the Staff Team of TXDPSRP, bring it up the CoC (Chain of Command), until it finally reaches the Head Administration.",
+"• If you believe there is a problem with a specific staff member, fill out the Staff Report form located on our forums, and we will take care of the issue, if necessary.",
+"• Always be honest with the Staff Team of TXDPSRP.",
```

Remove the comma on the last line of every list,
```suggestion
"• Always be honest with the Staff Team of TXDPSRP."
```